fvt/predictor/predictor_test.go

```diff
@@ -878,6 +878,8 @@ var _ = Describe("Predictor", func() {
 
 		AfterAll(func() {
 			FVTClientInstance.SetDefaultUserConfigMap()
+			By("Waiting for a stable deploy state after changing the user config")
+			WaitForStableActiveDeployState()
```

Comment:
I did not change the predictor_tests, but since those tests change the user config that is expected by other tests, I assume it was done to get back into the default state

Reply:
I only added the wait since I struggled to get stable deploys